cli/config.py

```diff
@@ -1,13 +1,26 @@
 import os
+
+from cli.consts import DEV_MODE_ENV_VAR_NAME
 from cli.utils.yaml_utils import read_file
 from cli.user_settings.configuration_manager import ConfigurationManager
 
-
 relative_path = os.path.dirname(__file__)
 config_file_path = os.path.join(relative_path, 'config.yaml')
 config = read_file(config_file_path)
 configuration_manager = ConfigurationManager()
+DEFAULT_CONFIGURATION = {
+    DEV_MODE_ENV_VAR_NAME: 'False'
+}
+configuration = dict(DEFAULT_CONFIGURATION, **os.environ)
```

Comment:
why you did not add it to the cyclient/config.py file?

Reply:
Fixed